Fix issue #352 (#523)
hsf-training/cpluspluscourse: talk/tools/formatting.tex

		\item \mintinline{bash}{clang-format -i <file.cpp>} (looks for .clang-format file)
		\item \mintinline{bash}{git clang-format} (formats local changes)
		\item \mintinline{bash}{git clang-format <ref>} (formats changes since git \textless{}ref\textgreater{})

> 		\item Some editors/IDEs (e.g. VSCode) find a .clang-format file and adapt

Reviewer: ```suggestion
		\item Most editors/IDEs can find a .clang-format file and adapt
```
